### ISSUE TITLE
Replay Log file wrong order

### ISSUE THREAD

**Issue by nico0481** [collin80/SavvyCAN#647]
I'm trying to replay a log file using "Playback" and it seems that sending start at second frame of the file. last replayed frame is the number 1.
Expected: playing frame1, frame2, frame3, frame4, frame5
Actually played is: frame2, frame3, frame4, frame5, frame1
I try by using "loopback" with virtualcan bus.
See attached .csv small log file (fake IDs and data for the test) and logwindow screenshot

[logtestorder.csv](https://github.com/collin80/SavvyCAN/files/11770958/logtestorder.csv)
![image](https://github.com/collin80/SavvyCAN/assets/45109152/05deae62-36d2-4c31-b77e-283493cf490f)

**Comment by collin80** [comment]
You're right. There is an off by one error in the sending code that skips the first frame. It does need to be fixed.
